### PR TITLE
Bugfix/lds selector does not change base lds url

### PR DESCRIPTION
**Pull request by takvamborgen**
* Change of LDS instance in user preferences now changes the actual LDS URL, not only the Zeppelin Instance (from be-workbench)

* Replaced  DataResource with StatisticalProgram

### CODE REVIEW

File: src/properties/properties-staging-bip.js [hunk 1]
```diff
@@ -2,15 +2,17 @@ import defaultProps from './properties-default'
...
+const baseUrl = "https://workbench.staging-bip-app.ssb.no/be/"
 
 export default {
   ...env,
   api: {
-    lds: ldsBaseUrl,
-    backend: "https://workbench.staging-bip-app.ssb.no/be/workbench-backend/",
-    role: ldsBaseUrl + "/ns/Role/",
-    dataResource: ldsBaseUrl + "/ns/StatisticalProgram/",
+    ldsOld: 'https://workbench.staging.ssbmod.net/be/lds-c',
```
Comment (bjornandre) [inline src/properties/properties-staging-bip.js]: Remove?

File: src/context/ContextProvider.js [hunk 2]
```diff
@@ -46,20 +47,41 @@ export class ContextProvider extends Component {
...
           user: {
             ...prevState.user,
             userPrefs: userPrefs
-          }
+          },
+          ldsService: Properties.mock.lds === true ? new LdsServiceMock()
+            : new LdsServiceImpl(ref.getLdsUrl(userPrefs.preferences.lds))
         }))
       }
+      // TODO could this be used for refreshing current page?
```
Comment (bjornandre) [inline src/context/ContextProvider.js]: Remove if the callback is not used. I suggest using hooks instead (and monitor changes in userPrefs)

Comment (takvamborgen) [reply src/context/ContextProvider.js]: I use the callback in App.js, which is still an old-style class, to update the state when user preferences are set.

File: src/properties/properties-staging-bip.js (hunk 1 above)
Comment (takvamborgen) [reply src/properties/properties-staging-bip.js]: The one that got away. I will fix.

**Pull request merged**